v20.2/spatial-data.md

```diff
@@ -0,0 +1,305 @@
+
+{{site.data.alerts.callout_info}}
+This documentation is for an early alpha release. Be prepared for CockroachDB's spatial support and this documentation to change before the 20.2 GA release.
+{{site.data.alerts.end}}
+
+## Compatibility
+
+Just as CockroachDB strives for [Postgres compatibility](postgresql-compatibility.html), our spatial data support is designed to be as compatible as possible with the functionality provided by the [PostGIS](https://postgis.net) extension.
+
+However, we do not yet implement the full list of PostGIS built-in functions and operators.  Also, our indexing works differently (see the [Performance](#performance) section below).  For a list of the spatial functions we support, see [Geospatial functions](functions-and-operators.html#geospatial-functions).
+
+If your application needs support for functions that are not yet implemented, please check out [our meta-issue for built-in function support on GitHub](https://github.com/cockroachdb/cockroach/issues/49203), which describes how to find an issue for the built-in function(s) you need.
+
+## Troubleshooting
+
+For general CockroachDB troubleshooting information, see [this troubleshooting overview](troubleshooting-overview.html).
+
+If you need help troubleshooting an issue with our spatial support, please get in touch using our [Support resources](support-resources.html).
+
+## Performance
+
+We recommend using `GEOMETRY` as much as possible for best performance, and converting to the `GEOGRAPHY` type only where needed by your application.
+
+In order to avoid full table scans, make sure to add indexes to any columns that are frequently accessed. You can check which queries are using which indexes using the [`EXPLAIN`](explain.html) statement. For more information about general query tuning (including index usage), see [Make queries fast](make-queries-fast.html).
+
+The syntax for adding an (inverted) index to a geometry column is:
+
+{% include copy-clipboard.html %}
+~~~ sql
+CREATE INDEX tornado_geom_idx ON tornadoes USING GIN (geom);
+~~~
+
+For more information about how inverted indexes work, see [Inverted indexes](inverted-indexes.html).
+
+Because CockroachDB is a scale-out, multi-node database, our spatial indexing strategy is based on a [space-filling curve](https://en.wikipedia.org/wiki/Space-filling_curve) design (also known as "divide the space"), rather than the [R-Tree](https://en.wikipedia.org/wiki/R-tree) data structure used by some other spatial databases (also known as "divide the objects"). Other databases that use a "divide the space" strategy include Microsoft SQL Server and MongoDB.
+
+We are still working on optimizing our performance for some common use cases. In most cases we expect to be within 2x of the performance of PostGIS for supported queries.
```

Comment:
I would probably support omitting this line at this point in time; we just have nowhere near enough benchmarks.